### ISSUE TITLE
`readonly` does not work as a property

### ISSUE THREAD

**Issue by walfie** [OvermindDL1/bucklescript-tea#131]
I tried using `Tea.Html2.Attributes.readonly` on an input field and expected the field to be unmodifiable, however that wasn't the case.

It seems like the issue might be that currently it's set as a property rather than an attribute:
https://github.com/OvermindDL1/bucklescript-tea/blob/94112931d5b9eaf7ae924d9b9162aef6f8530bcb/src-ocaml/tea_html2.ml#L339

It worked when I tried using `Vdom.attribute "" "readonly" "readonly"` instead. E.g.,
```ocaml
let view model =
  div
    []
    [ input' [ Tea.Html2.Attributes.readonly true; value "hello" ] []
    ; input' [ Vdom.attribute "" "readonly" "readonly" ; value "hello" ] []
    ]
```
The first input field is modifiable when it shouldn't be (in Chrome, at least).

As a sidenote, this property is only in `Html2` and is missing from `Html`, I'm not sure if that's intentional.

**Comment by OvermindDL1** [comment]
Oy, yeah that should be an attribute, correcting in next version.

Html2 follows Elm's 0.18+ API, Html follows Elm's pre 0.18 API, they fully work together though, just some naming conflicts at times.  :-)

**Comment by OvermindDL1** [comment]
Ah, it came in from a PR, might be others I need to look over too...  For now this one is fixed in, 0.15.0, sorry for the delay, Covid-19 stuff.  ^.^;

**Comment by walfie** [comment]
Thanks!
